pyproject.toml

```
@@ -58,3 +58,5 @@ documentation = "https://stravalib.readthedocs.io"
 repository = "https://github.com/stravalib/stravalib"
 changelog = "https://github.com/stravalib/stravalib/blob/master/changelog.md"
 
+[tool.black]
+line-length = 79
```

Maybe we can also add the isort black profile? I just ran into issues locally because these two made conflicting changes:

```suggestion
[tool.black]
line-length = 79

[tool.isort]
profile = "black"

```

---

oh i've heard about this. sure i actually haven't used isort before but i can see why it's super valuable. for some reason i can't just accept this so i'll fix locally.